tests/unit/s2n_session_ticket_test.c

```diff
@@ -594,7 +609,11 @@ int main(int argc, char **argv)
         /* Set session state lifetime for 15 hours which is equal to the default lifetime of a ticket key */
         EXPECT_SUCCESS(s2n_config_set_session_state_lifetime(server_config, S2N_SESSION_STATE_CONFIGURABLE_LIFETIME_IN_SECS));
 
-        EXPECT_SUCCESS(s2n_config_add_ticket_crypto_key(server_config, ticket_key_name1, s2n_array_len(ticket_key_name1), ticket_key1, s2n_array_len(ticket_key1), 0));
+        /* Add a session ticket key with an intro time in the past so that the key is immediately valid */
+        POSIX_GUARD(server_config->wall_clock(server_config->sys_clock_ctx, &now));
```

Comment:
Setting the wall_clock late here isn't as worrying as the previous example, but I'd still probably suggest setting it when you configure the config for all these tests. Just to avoid unexpected behavior.